pkgs/by-name/gh/ghbackup/package.nix

```diff
@@ -0,0 +1,45 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  git,
+  makeWrapper,
+}:
+
+buildGoModule rec {
+  pname = "ghbackup";
+  version = "0.0.1";
+
+  src = fetchFromGitHub {
+    owner = "voiapp";
+    repo = "ghbackup";
+    rev = "v${version}";
+    hash = "sha256-R7vtxxk0fzOMd99lT1dRkSJV3ENdItAUu01twjjGRxI=";
+  };
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  vendorHash = null;
+
+  postPatch = ''
+    # The source is a fork that has a patch applied,
+    # but didn't rename the import. Thus we need to init
+    # with the original package name.
```

Comment:
Maybe you can use the original source instead and apply the two patches from the fork on top

Reply:
Good idea! Applied